src/components/controlItems/ControlItems.js

```diff
@@ -0,0 +1,110 @@
+import { connect } from 'react-redux';
+import FormInput from '../formInput';
+import ButtonsActions from '../buttonsActions';
+
+import './controlItems.scss';
+
+class ControlItems extends Component {
+  state = {
+    originListCat: [],
+    listCat: [],
```

Comment:
you don't need to keep `originListCat` and `listCat`, you have it in props. 
Use `props.rightItems.filter(...)` in `render` or create some method for this `getFilterredItems() {...}`